libs/api/metadata-converter/src/lib/xml-utils.ts

```diff
@@ -448,3 +489,29 @@ export function renameElements(
   doReplace(rootElement)
   return rootElement
 }
+
+export function readGeometry(el: XmlElement): Geometry {
```

Comment:
these two functions (read/write geom) might be better of in the iso19139/utils directory in their own file (e.g. geometry.ts) because the xml utils are meant to be shared across any XML schema